android/src/main/java/com/polidea/blemulator/bridging/DartMethodCaller.java

```diff
@@ -445,4 +458,11 @@ public void notImplemented() {
                     }
                 });
     }
+
+    public void cancelTransaction(final String transactionId) {
+        Log.i(TAG, "cancelTransaction");
+        dartMethodChannel.invokeMethod(DartMethodName.CANCEL_TRANSACTION, new HashMap<String, String>() {{
+            put(SimulationArgumentName.TRANSACTION_ID, transactionId);
```

Comment:
sometime you use `ArgumentKey.TRANSACTION_ID` and sometimes `SimulationArgumentName.TRANSACTION_ID`.  Why?

Reply:
Both should ArgumentName.TRANSACTION_ID.
SimulationArgumentName is for arguments passed from Dart to platform, and ArgumentKey is from FlutterBleLib.